frontend/src/utils/api.ts

```diff
@@ -77,12 +77,7 @@ window.decodeHash = function decodeHash() {
     .forEach((param) => {
       const [key, val] = param.split('=');
       if (key) {
-        try {
-          hashParams[key] = decodeURIComponent(val);
-        } catch (error) {
-          (this as any).userUpdated = false;
-          (this as any).isLoading = false;
-        }
+        hashParams[key] = decodeURIComponent(val);
```

Comment:
Oh I moved it to the app component itself (https://github.com/reposense/RepoSense/pull/1914/commits/eaf1b7ada45b84a71e05620ee03d3bdeb9c22fe5) since the `this.userUpdated` was updating the data of that vue component, so I thought it would make sense to change it within the component itself vs in the global function (since the error will bubble up anyway).

I also removed the `.isLoading` as I don't think it is used anywhere.